x-pack/test/functional/es_archives/security_solution/timestamp_override_4/data.json

```diff
@@ -6,7 +6,7 @@
       "message": "hello world 4",
       "@timestamp": "2020-12-16T15:16:18.570Z",
       "event": {
-        "ingested": "2020-12-16T15:16:18.570Z"
+        "ingested": "2020-12-16T16:16:18.570Z"
```

Comment:
NOTE: Slight change just to ensure the tests still pass since `event.ingested` is usually a different value.